lib/scrabble.rb

```diff
@@ -15,4 +11,14 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+  def score(word)
+    if word == nil || word.empty?
+       word = 0
+    else
+      word.upcase.split("").map do |word|
+      point_values[word]
+    end.inject(:+)
```

Comment:
This works, but if you're using Ruby 2.4, you can just use `sum` which is a little more readable.